wpiunits/src/main/java/edu/wpi/first/units/Measure.java

```diff
@@ -88,8 +88,11 @@ default Measure<U> times(double multiplier) {
   @SuppressWarnings("unchecked")
   default <U2 extends Unit<U2>> Measure<?> times(Measure<U2> other) {
     if (other.unit() instanceof Dimensionless) {
-      // scalar multiplication
+      // scalar multiplication of this
       return times(other.baseUnitMagnitude());
+    } else if (unit() instanceof Dimensionless) {
```

Comment:
Going through tests for inverse and it seems with these changes taking a `Dimensionless` divided by a `Velocity<Dimensionless>` does not return a `Time` but instead a `Per<Time, Dimensionless>` and I think it's because of this line

Reply:
Good catch! There's actually two parts to this:

1. In my Dimensionless case of `divide()`, I'm not calling into `times()` at all, when I should.
2. `times()` needs to go through the "eliminate common units" portion of dimensional analysis before doing the dimensionless checks. I claim that on main right now, `Per<U, Dimensionless> * Dimensionless` does not return U because of this.